ledger/sandbox/src/main/resources/db/migration/h2database/V11__Command_deduplication.sql

```diff
@@ -0,0 +1,25 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+---------------------------------------------------------------------------------------------------
+-- V16: New command deduplication
+--
+-- Command deduplication has moved from ledger to participant
+---------------------------------------------------------------------------------------------------
+
+
+DROP INDEX idx_transactions_deduplication;
+
+
+CREATE TABLE participant_commands(
+  -- The deduplication key
+  deduplication_key  varchar primary key   not null,
+  -- The time the command was first submitted
+  submitted_at       timestamptz           not null,
+  -- The time the command will stop being deduplicated
+  ttl                timestamptz           not null,
+  -- If set, then the command was successfully submitted
+  success            boolean,
```

Comment:
Why is this nullable?

Reply:
`WriteService.submitTransaction` is async. While the submission is in progress, it is unknown whether it was successful.